### ISSUE TITLE
Elementor Pro Template Kit "Apply" Function Not Working

### ISSUE THREAD

**Issue by BillyBatson88** [elementor/elementor#15705]
After nearly a week of trying - and having others try - it appears that the new "Kit Library" of Template Kits cannot be utilized.  Every time I've tried, I get a "Something Went Wrong" error message.   When I refresh and try again, I get the "**Connect To The Elementor Template Library By Creating An Account**" dialogue box - even though I'm trying to use the Template Kit feature **WITH** my activated/verified/connected Elementor Pro License.

When I do click the "Connect ..." button, I am brought to a screen saying that "**Your License Has Exceeded The Limitation**."

The problem is, **HOW** can my license have "exceeded its limitation" **if I'm trying to utilize a Template Kit via my active/verified/connected Elementor Pro License within the site backend for which the license is assigned?**

Has anyone else been able to "Apply" any of the Template Kit Templates - as advertised by Elementor as part of the latest Pro update ???

**If not, then this is definitely a serious bug that needs to be fixed.** 

It wouldn’t bother me so much if there wasn’t one of the templates with “ready-made/configured” features I REALLY LIKE and REALLY want to utilize - and being able to would save me A LOT of time trying to clone design/configure it myself. 

 I wish there was a way the Elementor Support could verify my valid Elementor Pro license and provide a downloadable zip file with the particular template kit I want/need.

**Comment by shilo-ey** [comment]
Please contact our support at support @elementor.com for guidance in resolving this issue and mention that you were directed by me and reference this issue.

Thanks.

**Comment by BillyBatson88** [comment]
I did as you directed and the Elementor Support Team has not seen fit to respond relative to this ongoing issue - despite my advising of your direction.

Further, I updated my Elementor Pro to the latest 3.3.3 update - and the issue continues to persist . . . still getting the "_Something went wrong. - Nothing to worry about, just try again. If the problem continues, head over to the Help Center_." error message and unable to apply any Template Kit (let alone the ONE I very much want/need).

Would it be at all possible to obtain a zip file of the ONE Template Kit I want/need- while this persistent problem continues?
The Template Kit to which I am referring is the black "MUSIC FESTIVAL" Kit.

Looking forward to your further pro-active response.

THANK YOU !!!
